source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst

```diff
@@ -8,11 +8,11 @@ to start Linux development on the TI microprocessor.
 
 **The** |__SDK_FULL_NAME__| **Release supports installer only for Linux machine.**
 
-+--------------------------------------------+----------------------------------------------------------------------+
-| **Installer Details**                      | **Installers Name**                                                  |
-+--------------------------------------------+----------------------------------------------------------------------+
-| PSDK Linux AM62A Installer (Linux version) | ti-processor-sdk-linux-am62axx-evm-<version>-Linux-x86-Install.bin   |
-+--------------------------------------------+----------------------------------------------------------------------+
++--------------------------------------------+-----------------------------------------------------------------------------+
+| **Installer Details**                      | **Installers Name**                                                         |
++--------------------------------------------+-----------------------------------------------------------------------------+
+| PSDK Linux AM62A Installer (Linux version) | ti-processor-sdk-linux-edgeai-am62a-evm-<version>-Linux-x86-Install.bin     |
```

Comment:
Any reasons to not use "|\_\_SDK_INSTALL_FILE\_\_|" which you are already using in this file?

https://github.com/TexasInstruments/processor-sdk-doc/blob/master/configs/AM62AX/AM62AX_linux_config.txt#L22

Reply:
I was not aware of this, using |__SDK_INSTALL_FILE__| as a replacement variable under Installer Name